docs/how-to/service-placement.rst

```diff
@@ -0,0 +1,71 @@
+Start Ceph services on any node of the MicroCeph cluster
+========================================================
+
+MicroCeph allows users to start supported ceph service on any target host in the MicroCeph cluster. This can be achieved by providing the remote hostname as the --target parameter to the enable command (as described below)
+
+.. list-table:: Supported Ceph Services
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Name
+     - Description
+   * - mon
+     - Ceph Monitor Service
+   * - mgr
+     - Ceph Manager Service
+   * - mds
+     - Ceph Monitor Service
+   * - rgw
+     - Ceph Radosgw Service
+
+The enable command is used as follows:
+  .. code-block:: shell
+
+    $ sudo microceph enable <service> --target <node_hostname> ...
+
+.. note:: Some services can be provided with additional parameters while enabling them, these can be listed for any service by issuing respective help command.
+
+  .. code-block:: shell
+
+    $ sudo microceph enable <service> --help
+
+1. Enable RadosGW service:
+
+  .. code-block:: shell
+
+    $ sudo microceph status
+    MicroCeph deployment summary:
+    - node1-2c3eb41e-14e8-465d-9877-df36f5d80922 (10.111.153.78)
+      Services: mds, mgr, mon, osd
+      Disks: 3
+    - workbook (192.168.29.152)
+      Services: mds, mgr, mon
+      Disks: 0
+    $ sudo microceph enable rgw --target node1 --port 8080 
+    $ sudo microceph status
+    MicroCeph deployment summary:
+    - node1 (10.111.153.78)
+      Services: mds, mgr, mon, rgw, osd
+      Disks: 3
+    - workbook (192.168.29.152)
+      Services: mds, mgr, mon
+      Disks: 0
+
+2. Enable Mon Service:
```

Comment:
Nit: suggest to keep service lowercase like the RadosGW entry above (here and below)

Reply:
ackl